hubs/templates/hubs.html

```diff
@@ -92,4 +98,19 @@
     });
 });
 </script>
+<script>
+$(document).ready(function (){
+    $('#mozinstall').click(function() {
+        navigator.mozApps.install("http://127.0.0.1:8000/static/manifest.webapp");
```

Comment:
Same here, we can't hardcode this URL.  But, didn't this code appear somewhere else in the pull request also?